app/config.py

```diff
@@ -19,11 +19,15 @@
 APIs Config
```

Comment:
The get function isn't a method of environ but rather a method used on any dictionary. Environ is merely a dictionary of environment variables. This leads the following exciting possibility...default values. 

The second argument in a get function is the default value: so you don't have to use "or" anymore.

So:

`environ.get("FUND_STORE_API_HOST") or TEST_FUND_STORE_API_HOST`

becomes 

`environ.get("FUND_STORE_API_HOST", TEST_FUND_STORE_API_HOST)`

Reply:
Ah-ha...! Yes get() does provide a default option... But the current expression, I propose is (marginally) preferable as a slightly more strict default - as it returns the default also when "FUND_STORE_API_HOST" is set, but to an unusable empty string, or other falsy value.

Although this will undoubtably create arguments/wars in the community on whether an empty string should be considered false.. obv!

DEMO
`environ = {"test": ""}`
`print(environ.get("test") or "nice fallback")`
`print(environ.get("test","might not get the fallback"))`

Reply:
got to go with the OR operand solution (i agree we want to use the default for falsy's) with Jacks nice bracket formatting suggestion to improve readability :)

Reply:
for env vars where we can guarantee the fallback/default value is suitable *